### PR TITLE
Add haxe parser from Geany

### PR DESCRIPTION
**Pull request by techee**
Haxe <https://en.wikipedia.org/wiki/Haxe>

### CODE REVIEW

File: Units/parser-qemuhx.r/enable.d/args.ctags [hunk 1]
```diff
@@ -1,4 +1,4 @@
 --sort=no
 --extras-QemuHX=+{funcmap}
 --fields=+E
-
+--language-force=QemuHX
```
Comment (masatake) [inline Units/parser-qemuhx.r/enable.d/args.ctags:4]: Though this should be fixed, I would like you to delete this modification from this commit?
Can I ask you?

File: parsers/haxe.c [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+	 */
+	def->kindTable          = HxKinds;
+	def->kindCount  = ARRAY_SIZE (HxKinds);
+	def->parser             = findHxTags;
+	/*def->initialize = initialize;*/
+	return def;
+}
+
+
+
```
Comment (masatake) [inline parsers/haxe.c:220]: Could you remove these empty lines at the end of file?

File: Units/parser-qemuhx.r/enable.d/args.ctags (hunk 1 above)
Comment (techee) [reply Units/parser-qemuhx.r/enable.d/args.ctags:4]: Hmm, not really, the test fails without this

Comment (masatake) [reply Units/parser-qemuhx.r/enable.d/args.ctags:4]: Oh, I see why this change is needed.
Ignore what I wrote.

Comment (masatake) [reply Units/parser-qemuhx.r/enable.d/args.ctags:4]: I have to provide a command-line interface to define an arbitrator resolving this kind of conflict in the future

**Comment by masatake** [comment]
Thank you.
I'm sorry to be late for work on this parser.

**Comment by techee** [comment]
> I'm sorry to be late for work on this parser.

Don't worry, no rush, we are still 1.5 years behind universal ctags in Geany ;-).

**Pull request merged**